src/Commands/Self/Plugin/InstallCommand.php

```diff
@@ -119,24 +119,48 @@ public function validate(CommandData $commandData)
      *
      * @throws \Pantheon\Terminus\Exceptions\TerminusException
      */
-    protected function convertPathProjects(array $projects): array
+    protected function convertRepositoryProjects(array $projects): array
     {
         $convertedProjects = [];
 
         foreach ($projects as $projectNameOrPath) {
-            if (!$this->hasProjectAtPath($projectNameOrPath)) {
-                // No project was found, presume the parameter is a project, and it has no path.
-                $convertedProjects[$this->getComposerProjectName($projectNameOrPath)] = null;
-            } else {
+            if ($this->isGitRepo($projectNameOrPath)) {
+                $pluginsDir = $this->getConfig()->get('plugins_dir');
+                $pluginFolderName = $this->createRandomDir('plugin-', $pluginsDir);
```

Comment:
@greg-1-anderson actually it's going to $HOME/.terminus/plugins-3.x/plugin-<random>

Maybe you ran the command from terminus dir and it got changed because of env vars using direnv?

Please confirm and I'll merge this PR

Reply:
kk I will check again; however, we don't need the random hash, do we? i.e. if we install it twice, it should go to the same place every time. We don't need multiple copies; overwrite is preferable.

Reply:
I am still seeing it installed to `/private/var/folders/wt/g2rxs8pj74g336k69gt4bz3r0000gp/T/plugin-A1dbEl`, even though I haven't set TERMINUS_HOME or anything of that nature.
```
  key: plugins_dir_basename
  env: TERMINUS_PLUGINS_DIR_BASENAME
  value: plugins-3.x
  source: /Users/ga/Code/pantheon/Alchemy/terminus/config/constants.yml
-
  key: base_dir
  env: TERMINUS_BASE_DIR
  value: /Users/ga/.terminus
  source: /Users/ga/Code/pantheon/Alchemy/terminus/config/constants.yml
-
  key: plugins_dir
  env: TERMINUS_PLUGINS_DIR
  value: /Users/ga/.terminus/plugins-3.x
  source: /Users/ga/Code/pantheon/Alchemy/terminus/config/constants.yml
```
I'll check the code further and see if there's some other variable affecting my base dir.

Reply:
The install location is taken from `createRandomDir` which is placed in tmp, not in $pluginsDir. If you simply do not make a random dir, then the install location should go to the right place.

Reply:
That's weird because it was supposed to use pluginsDir as the base dir. Anyway, I'm working on that change and it should be ready soon :)